.travis.yml

```diff
@@ -0,0 +1 @@
+language: rust
```

Comment:
you'll need a `rust: nightly` here too, as doxidize only runs on nightly

also an `os: - linux - osx` might be nice so we get both of them